src/main/java/net/mcreator/minecraft/MinecraftImageGenerator.java

```diff
@@ -703,8 +703,10 @@ public static BufferedImage generateDimensionPreviewPicture(Workspace workspace,
 				graphics2D.drawImage(tex, 8, 17, 6, 6, null);
 				graphics2D.drawImage(tex, 14, 17, 6, 6, null);
 
-				BufferedImage igniter = ImageUtils.autoCropTile(ImageIO.read(triggerTexture));
-				graphics2D.drawImage(igniter, 2, 2, 24, 24, null);
+				if (triggerTexture.isFile()) {
```

Comment:
trigger texture will exist even if trigger item is disabled. Pass triggerEnabled to this method to properly handle this

Reply:
Done!